providers/elasticsearch/src/airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -358,12 +371,31 @@ def _read(
 
         # If we hit the end of the log, remove the actual end_of_log message
         # to prevent it from showing in the UI.
-        def concat_logs(hits: list[Hit]):
+        def concat_logs(hits: list[Hit]) -> str:
             log_range = (len(hits) - 1) if hits[-1].message == self.end_of_log_mark else len(hits)
             return "\n".join(self._format_msg(hits[i]) for i in range(log_range))
 
         if logs_by_host:
-            message = [(host, concat_logs(hits)) for host, hits in logs_by_host.items()]
+            if AIRFLOW_V_3_0_PLUS:
+                from airflow.utils.log.file_task_handler import StructuredLogMessage
+
+                header = [
+                    StructuredLogMessage.model_construct(
```

Comment:
Why are we skipping the validation whe instanciating `StructuredLogMessage` ?

Reply:
This basically follows https://github.com/apache/airflow/pull/46827/files#diff-e7f34f73940eb52d92bb991abedc1c963431c5373c12dff739c8fb7d03e93d3aR417. Would like to confirm with @ashb whether there's any reason we're doing it. it not we probably could go with `StructuredLogMessage(...)`?

Reply:
I guess https://github.com/apache/airflow/pull/46827/files#diff-e6117e0d0a37880e15b0615aa1f1f936f7e85b61e915e68f2ca0c56073f79a92R28 might be the reason?

Reply:
@pierrejeambrun WDYT? Should we merge it, or should we change this one? Thanks!

Reply:
Likely not required, I was just thinking "why bother running validation if we know we don't need to". Happy to change it either way

Reply:
@pierrejeambrun would like to confirm whether we want to make this change. Either way works for me. I can make this change later 🙂

Reply:
Not strong about it. I think changing it doesn't hurt and is safer

Reply:
Got it. Sounds good. 👍  Will continue work on it.

Reply:
@pierrejeambrun just updated it. could you please take a look if you have some bandwidth? Many thanks!